packages/core/src/render3/l_node_static.ts

```diff
@@ -0,0 +1,132 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {DirectiveDef} from './public_interfaces';
+
+/** The type of the global ngStaticData array. */
+export type NgStaticData = (LNodeStatic | DirectiveDef<any> | null)[];
+
+/**
+ * LNode binding data (flywiehgt) for a particular node that is shared between all templates
```

Comment:
`flywiehgt` -> `flyweight`

Might be good to add a link to https://en.wikipedia.org/wiki/Flyweight_pattern while you're at it

Reply:
Oh jeez, how did that mangling happen haha? Thanks for catching.